pyccel/codegen/printing/ccode.py

```diff
@@ -288,6 +288,32 @@ def _traverse_matrix_indices(self, mat):
         rows, cols = mat.shape
         return ((i, j) for i in range(rows) for j in range(cols))
 
+    #========================== Numpy Elements ===============================#
+    def print_NumpyArange(self, expr, lhs):
+        start  = self._print(expr.start)
+        stop   = self._print(expr.stop)
+        step   = self._print(expr.step)
+        dtype  = self.find_in_ndarray_type_registry(self._print(expr.dtype), expr.precision)
+
+        target = Variable(expr.dtype, name =  self._parser.get_new_name('s'))
+        index  = ValuedVariable(NativeInteger(), name = self._parser.get_new_name('i'), value=0)
```

Comment:
Why do you need a `ValuedVariable`? These are usually used for arguments with a default value not assignments